app/Domain/Tickets/Templates/showAll.blade.php

```diff
@@ -215,20 +215,19 @@ class="ticketDropdown statusDropdown colorized show {{ $class }} f-left"
                                 </x-global::elements.table.cell>
             
                                 <x-global::elements.table.cell data-order="{{ $milestoneHeadline }}">
-                                    <x-global::forms._archive.dropdownPill
-                                        class="ticketDropdown milestoneDropdown colorized show label-default milestone f-left"
-                                        id="{{ $milestoneDropdownId }}" 
-                                        :style="'background-color:' . $milestoneColor" 
+                                    <x-global::forms.chip
+                                        class="ticketDropdown milestoneDropdown colorized milestone show f-left"
```

Comment:
Do we need those classes still? Or is this something we can move into a chip component css so that it's less tied to "tickets" and more generally applicable